src/phantom.js

```diff
@@ -82,7 +82,7 @@ function ResourceHandler(htmlroot, isWindows, resolve) {
             }
             networkRequest.changeUrl(url);
         } else if (ignoreRequests.test(url)) {
-            networkRequest.abort();
+            networkRequest.cancel();
```

Comment:
I cannot find any documentation about a cancel() method, where did you see this? Even looking into PhantomJS, there is no mention of a cancel():
https://github.com/ariya/phantomjs/blob/e024f31d662558eb62861b4c0d4f3869511b6983/src/networkaccessmanager.h#L55

However, I can find this page https://github.com/amir20/phantomjs-node#pageon, which seems to explain why abort() might not work, and what should be done to access it (runOnPhantom=true).